plugins/catalog-node/src/extensions.ts

```diff
@@ -40,3 +41,20 @@ export const catalogProcessingExtensionPoint =
   createExtensionPoint<CatalogProcessingExtensionPoint>({
     id: 'catalog.processing',
   });
+
+/**
+ * @alpha
+ */
+export interface CatalogAnalysisExtensionPoint {
+  addLocationAnalyzers(
```

Comment:
nit: the other extension point did singular wording here (would have been without the s) even though it's technically correct that it takes multiple - essentially, it was probably a mistake to make it possible to add more than one at a time, the convenience factor is microscopic and it just complicates things. So if this were singular and only accepted a single analyzer, it'd be a movement in the right direction probably

Reply:
sure - updated!